bot/code_review_bot/tasks/docupload.py

```diff
@@ -1,23 +1,58 @@
 # -*- coding: utf-8 -*-
+import os
+
 import structlog
 
 from code_review_bot.tasks.base import NoticeTask
 
 logger = structlog.get_logger(__name__)
 
+MAX_LINKS = 20
+
+DOC_LINK = """
+- file [{path}]({doc_url})
+"""
+COMMENT_LINKS_TO_DOC = """
+NOTE: {nb_docs_hint} modified in diff {diff_id}
+
+{pronoun} can be previewed for one week:{doc_urls}
+"""
+
 COMMENT_LINK_TO_DOC = """
-NOTE: Documentation was modified in diff {diff_id}
+NOTE: Several documentation files were modified in diff {diff_id}
 
-It can be previewed [here]({doc_url}) for one week.
+They can be previewed [here]({doc_url}) for one week.
 """
 
 
+def direct_doc_url(path, doc_url, trees):
+    root_doc_url = os.path.dirname(doc_url)
+    filename, _ = os.path.splitext(os.path.basename(path))
+    dir = os.path.dirname(path)
+    for doc_path, dir_path in trees.items():
```

Comment:
Nit: could you give more meaningful names to differentiate these variables more easily, so the following code is easier to read

Reply:
Did my best but if you have better names in mind, feel free to request changes again :stuck_out_tongue: